eclipse/src/saros/ui/eventhandler/ConnectingFailureHandler.java

```diff
@@ -64,15 +64,17 @@ private void handleConnectionFailed(Exception exception) {
         return;
       }
 
-      /* FIXME the active/default account is not always the account that is currently used for connecting */
       if (DialogUtils.popUpYesNoQuestion(
           "Connecting Error",
           generateHumanReadableErrorMessage((XMPPException) exception),
           false)) {
 
-        if (WizardUtils.openEditXMPPAccountWizard(accountStore.getActiveAccount()) == null) return;
+        /* FIXME the active/default account might not always be the account that is currently used for connecting */
+        final XMPPAccount accountUsedDuringConnection = accountStore.getDefaultAccount();
 
-        final XMPPAccount account = accountStore.getActiveAccount();
+        if (WizardUtils.openEditXMPPAccountWizard(accountUsedDuringConnection) == null) return;
```

Comment:
Throws assertion error when `accountUsedDuringConnection` is `null`: See [EditXMPPAccountWizard l.27](https://github.com/saros-project/saros/blob/master/eclipse/src/saros/ui/wizards/EditXMPPAccountWizard.java#L27)

Reply:
Was misguided by comment on line 43. Yeah I will add null check in the XMPPConnectionSupport and abort the connection attempt.

Reply:
This is actually already fixed in a follow up.

Reply:
For future occurrences, it would be useful to also provide a link to the followup PR.

Reply:
This is another wizard. Seems we have to many wizards ...

The ConfigurationWizard is the initial wizard where you can add/create an XMPPAccount, then choose your session color, etc.

This here is the EditXMPPAccountWizard and this have to be opened.

Reply:
My bad, I am totally tired. I had a hard time to understand what you want me to do ... sorry. Yeah I will add a link next time to make reviewing easier.